js/graviex.js

```diff
@@ -0,0 +1,1058 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ArgumentsRequired, OrderNotFound } = require ('./base/errors');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class graviex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'graviex',
+            'name': 'Graviex',
+            'version': 'v3',
+            'countries': [ 'MT', 'RU' ],
+            'rateLimit': 1000,
+            'has': {
```

Comment:
I found some missing implementations such as:

- fetchTime (https://graviex.net/documents/api_v3#!/timestamp/GET_timestamp_format)
- createDepositAddress (https://graviex.net/documents/api_v3#!/gen_deposit_address)
- ...

Please check that out 🙂

Reply:
@carlosmiei, Added these two missing endpoints and included them in UNIFIED METHOD as well. Please check-in latest commit https://github.com/ccxt/ccxt/commit/bb2b0b2ef29c557eee9dad97fee9c010de932c9a.